### ISSUE TITLE
Copy-S3Object changing variables outside its module scope

### ISSUE THREAD

**Issue by Matrikss** [aws/aws-tools-for-powershell#35]
Consider the following code:
``` powershell
$Version = 'myversion' 
Write-Output "VERSION: $Version" 
$s3Object = Copy-S3Object -BucketName $S3AWSBucketName -Key $S3AWSDirectory/$S3FileName -LocalFolder $env:temp -AccessKey $S3AWSAccessKeyId -SecretKey $S3AWSSecretAccessKey -Force
Write-Output "VERSION: $Version"
```

**Expected Behavior**
VERSION: myversion
VERSION: myversion

**Current Behavior**
VERSION: myversion
VERSION: 

**Steps to Reproduce (for bugs)**
Put the code above on a .ps1 file and execute in a new Powershell console.
The first execution will give the above behavior, the subsequent ones don't suffer from it.
The command needs to execute correctly, doesn't show the above behavior in case of failure.

**Context**
This broke our Infrastructure as Code framework and caused a very strange debug session to identify the cause.

**Your Environment**
* Affected module: AWSPowerShell
* Module version as reported by `Get-AWSPowerShellVersion`: 3.3.563.0
* Operating System and version as reported by `$PSVersionTable.OS`: empty
* PowerShell verions as reported by `$PSVersionTable.PSEdition`: Desktop

**Comment by matteo-prosperi** [comment]
Hello,
thanks a reporting this!
This is an unwanted side effect of a `ScriptsToProcess` we added in `3.3.563.0`. We are planning to release a fix for this issue today.

**Comment by matteo-prosperi** [comment]
The issue is resolved in the new version `3.3.563.1`. Thank you for reporting it.
